runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -512,6 +512,10 @@ task validatesRunnerV2Streaming {
       'org.apache.beam.sdk.transforms.CombineTest$WindowingTests.testSessionsCombineWithContext',
       'org.apache.beam.sdk.transforms.CombineTest$WindowingTests.testSlidingWindowsCombineWithContext',
 
+      // TODO(BEAM-13961) Investigation needed.
```

Comment:
I actually had it open before in #16859 too

Reply:
Thanks! I didn't realize it :)